### ISSUE TITLE
[feature req] Make storage information available.

### ISSUE THREAD

**Issue by tomtom13** [travisghansen/hass-pfsense#186]
Hi, this might be a bit complicated, however I think it would be good to expose details about pfsense underlying storage for monitoring. 

My pfSense is running on raid_1 ZFS, and recently I've discovered that one of the drives decided to die. It's not that much of a problem since zfs is pretty resilient however I think that people might get some mileage from warnings about underlying storage issues.

**Comment by travisghansen** [comment]
Yeah, seems like a good idea. I think I have sensors for space usage but pool health is a good idea as well. My time is limited on this project atm but we can leave the issue open and take a look when time arrives.
